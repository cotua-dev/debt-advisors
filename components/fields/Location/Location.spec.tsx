import { render, RenderResult } from '@testing-library/react';
import { Location } from './index';

describe('Location Component', (): void => {
    let renderResult: RenderResult;

    beforeAll((): RenderResult => renderResult = render(<Location/>));

    test('renders', (): void => {
        const { container } = renderResult;
        expect(container.firstChild).toMatchSnapshot();
    });

    afterAll((): void => renderResult.unmount());
});
