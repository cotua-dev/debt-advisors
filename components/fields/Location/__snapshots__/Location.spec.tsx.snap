// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location Component renders 1`] = `
<div
  class="location-wrapper"
>
  <input
    class="location-field"
    id="zip-code"
    name="zip-code"
    type="text"
    value=""
  />
  <picture>
    <source />
    <img
      alt="Powered By Google Logo"
      class="powered-by-google-logo"
      loading="lazy"
    />
  </picture>
</div>
`;
